net/mwan3/files/etc/mwan3.user

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+#
+# This file is interpreted as shell script.
+# Put your custom mwan3 action here, they will
+# be executed with each netifd hotplug interface event.
```

Comment:
You could maybe clarify the message text in user script a bit so that users really get it right. If I read the hotplug script correctly, this user script is executed on "each netifd hotplug interface event **on interfaces for which mwan3 is enabled**". 

Not quite sure about my interpretation, but that is my reading of the hotplug script's action.